core/assets/bundles/bundle.properties

```diff
@@ -1076,6 +1078,8 @@ block.copper-wall.description = A cheap defensive block.\nUseful for protecting
 block.copper-wall-large.description = A cheap defensive block.\nUseful for protecting the core and turrets in the first few waves.\nSpans multiple tiles.
 block.titanium-wall.description = A moderately strong defensive block.\nProvides moderate protection from enemies.
 block.titanium-wall-large.description = A moderately strong defensive block.\nProvides moderate protection from enemies.\nSpans multiple tiles.
+block.plastanium-wall.description = Prevents automatic wire connections.\nmanual connections remain unimpeded.
```

Comment:
One last thing: how about
`A special type of wall that absorbs electric arcs.\nBlocks automatic power node connections. Manual connections remain unimpeded.`

...and likewise for the large wall.

Reply:
Looks a bit out of place though, are you sure? 😗 
![Screen Shot 2019-10-27 at 21 27 33](https://user-images.githubusercontent.com/3179271/67641194-b2683200-f900-11e9-95b3-0f5beb093771.png)
![Screen Shot 2019-10-27 at 21 27 40](https://user-images.githubusercontent.com/3179271/67641195-b2683200-f900-11e9-86ce-581a70e9d324.png)

Reply:
How about just `A special type of wall that absorbs electric arcs and blocks automatic power node connections`

Reply:
I tried to edit it in just now but github mobile doesn't want to cooperate, I'll try it tomorrow from my desktop, but feel free to merge & edit, or even edit the pull itself, i believe i had the checkbox on that allowed edits from maintainers 🤔